presto-native-execution/scripts/release-centos-dockerfile/opt/entrypoint.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -eExv -o functrace
+
+SCRIPT_DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")
+PRESTO_HOME="${PRESTO_HOME:-"/opt/presto"}"
+USE_ENV_PARAMS=${USE_ENV_PARAMS:-0}
+
+source "${SCRIPT_DIR}/common.sh"
+
+trap 'exit 2' SIGSTOP SIGINT SIGTERM SIGQUIT
+trap 'failure "LINENO" "BASH_LINENO" "${BASH_COMMAND}" "${?}"; [ -z "${DEBUG}" ] && exit 1 || sleep 3600' ERR
+
+if [[ "${DEBUG}" == "0" || "${DEBUG}" == "false" || "${DEBUG}" == "False" ]]; then DEBUG=""; fi
+
+http_server_port=8080
+discovery_uri="http://127.0.0.1:${http_server_port}"
+
+while getopts ':-:' optchar; do
+  case "$optchar" in
+    -)
+      case "$OPTARG" in
+        discovery-uri=*) discovery_uri="${OPTARG#*=}" ;;
+        http-server-port=*) http_server_port="${OPTARG#*=}" ;;
+        use-env-params) USE_ENV_PARAMS=1 ;;
+        *)
+          presto_args+=($optchar)
+          ;;
+      esac
+      ;;
+    *)
+      presto_args+=($optchar)
+      ;;
+  esac
+done
+
+function node_command_line_config()
+{
+  printf "presto.version=0.273.3\n"                    >  "${PRESTO_HOME}/config.properties"
+  printf "discovery.uri=${discovery_uri}\n"            >> "${PRESTO_HOME}/config.properties"
+  printf "http-server.http.port=${http_server_port}\n" >> "${PRESTO_HOME}/config.properties"
+
+  printf "node.environment=intel-poland\n" >  "${PRESTO_HOME}/node.properties"
+  printf "node.location=torun-cluster\n"  >> "${PRESTO_HOME}/node.properties"
+  printf "node.id=${NODE_UUID}\n"        >> "${PRESTO_HOME}/node.properties"
+  printf "node.ip=$(hostname -I)\n"      >> "${PRESTO_HOME}/node.properties"
+}
+
+function node_configuration()
+{
+  cat "${PRESTO_HOME}/config.properties.template" > "${PRESTO_HOME}/config.properties"
+  cat "${PRESTO_HOME}/node.properties.template" > "${PRESTO_HOME}/node.properties"
```

Comment:
ok, I think it was failing to find because this is copied on the Dockerfile under `/opt/presto/etc` not under `/opt/presto`, right?
```suggestion
  cat "${PRESTO_HOME}/etc/config.properties.template" > "${PRESTO_HOME}/config.properties"
  cat "${PRESTO_HOME}/etc/node.properties.template" > "${PRESTO_HOME}/node.properties"
```

Reply:
I have fixed this to run with defaults if no config is attached. The above suggestion is not valid to this workflow - where use should mount config templates to presto home dir.

Reply:
https://github.com/prestodb/presto/pull/18572/commits/23c0784c17115047f0d67af901f5360ee1190171